### ISSUE TITLE
Category toggles missing from tab order

### ISSUE THREAD

**Issue by jacobtylerwalls** [azavea/green-equity-demo#101]
You should be able to tab through the category buttons with the TAB button on the keyboard. Currently the tab order skips from the Element 84 link to the state tooltips.
